web/modules/custom/dpl_library_agency/dpl_library_agency.permissions.yml

```diff
@@ -0,0 +1,3 @@
+administer library agency configuration:
+  title: 'Administer library agency configuration'
+  description: 'Read and view access for the library agency configuration.'
```

Comment:
Please rewrite this description. "Read and view access" does not seem very telling for the scope of the module.